dashboard/test/ui/runner.rb

```diff
@@ -51,18 +51,22 @@
   opts.on("-f", "--feature Feature", Array, "Single feature or comma separated list of features to run") do |f|
     $options.feature = f
   end
-  opts.on("-p", "--pegasus Domain", String, "Specify an override domain for code.org, e.g. localhost:9393") do |d|
-    $options.pegasus_domain = d
+  opts.on("-l", "--local", "Use local webdriver, not Saucelabs") do
```

Comment:
Might also add a little note about this to the readme https://github.com/code-dot-org/code-dot-org/tree/staging/dashboard/test/ui